mlir/tools/mlir-clang/Test/Verification/lower_to.c

```diff
@@ -0,0 +1,9 @@
+// RUN: mlir-clang %s --function=foo | FileCheck %s
+
+#pragma lower_to(bar, "std.addf")
```

Comment:
Can we use "lower_to" also on nested loops or CallOp only?

Reply:
Possibly yes. I think it is more like an implementation issue here. I only allow lower_to on function mainly because function has a similar interface as an MLIR operation. If we want to apply lower_to on arbitrary language structure, we might need to figure out how to decide the parameter list for the operation, etc.